src/components/ProjectSettingsTabs/AliasRow.tsx

```diff
@@ -0,0 +1,159 @@
+import { useState } from "react";
+import styled from "@emotion/styled";
+import {
+  SegmentedControl,
+  SegmentedControlOption,
+} from "@leafygreen-ui/segmented-control";
+import { Subtitle } from "@leafygreen-ui/typography";
+import { Accordion } from "components/Accordion";
+import { SpruceFormProps } from "components/SpruceForm";
+
+const VariantInput: Record<string, string> = {
+  Regex: "variant",
+  Tags: "variantTags",
+} as const;
+
+const TaskInput: Record<string, string> = {
+  Regex: "task",
+  Tags: "taskTags",
+} as const;
+
+// Extract index of the current field via its ID
+const getIndex = (id: string): number => {
+  if (!id) return null;
+
+  const stringIndex = id.substring(id.lastIndexOf("_") + 1);
+  const index = Number(stringIndex);
+  return Number.isInteger(index) ? index : null;
+};
+
+export const AliasRow: SpruceFormProps["ObjectFieldTemplate"] = ({
+  disabled,
+  formData,
+  idSchema,
+  properties,
+  readonly,
+  uiSchema,
+}) => {
+  const [, alias, variant, variantTags, task, taskTags] = properties;
+  const isDisabled = disabled || readonly;
+  const accordionTitle = uiSchema["ui:accordionTitle"];
+
+  const index = getIndex(idSchema.$id);
+
+  const [variantInput, setVariantInput] = useState(
+    formData?.variant ? VariantInput.Regex : VariantInput.Tags
+  );
+  const [taskInput, setTaskInput] = useState(
+    formData?.variant ? TaskInput.Regex : TaskInput.Tags
+  );
+
+  // Clear form when toggling between segmented control options
+  const clearForm = (selectedSegment: string) => {
+    switch (selectedSegment) {
+      case VariantInput.Regex:
+        variantTags.content.props.onChange([]);
+        break;
+      case VariantInput.Tags:
+        variant.content.props.onChange("");
+        break;
+      case TaskInput.Regex:
+        taskTags.content.props.onChange([]);
+        break;
+      case TaskInput.Tags:
+        task.content.props.onChange("");
+        break;
+      default:
+    }
+  };
+
+  return (
```

Comment:
Would it make sense to default to the `Tags` tab if the regex field is empty? (At least for this page, all of the `SegmentedControl`s default to the `Regex` tab and I was kind of confused why it was empty until I clicked on it)

https://user-images.githubusercontent.com/47064971/151233514-db5add5a-d4c0-4516-8db5-4bf312480162.mov